hack/build.sh

```diff
@@ -323,6 +355,7 @@ Usage: $(basename $BASH_SOURCE) [... options ...]
 with the following options:
 
 -f  --fast                    Only compile (without dep update, formatting, testing, doc gen)
+-p  --platform                Specify the target platform for cross-compilation (e.g., linux-amd64, darwin-amd64)"
```

Comment:
Should the example be without dashes? Given the function takes `ARGS[1]` `ARGS[2]`?
```suggestion
-p  --platform                Specify the target platform for cross-compilation (e.g., linux amd64, darwin amd64)"
```